README.md

```diff
@@ -22,8 +22,8 @@ Disclaimer - this software has no association to `Corsair Gaming, Inc` , is prov
 		<add key="HeadsetName" value="Headphones"/> //Use your headphone's name
 		<add key="SpeakersName" value="Speakers"/> //if you have more than one device named "Speakers"(for example if you use steam remote play you might heve a device named "Steam Streaming Speakers", which will too show up in the command output), then use the full name of the device as reported by get-audiodevice -list | where Type -eq 'Playback' in powershell
 		<add key="Autorun" value="true"/>
-		<add key="ConnectedRegexOverride" value="\d{4}\-\d\d\-\d\d\s(?:\d\d\:)*\d\d.\d+ I cue\.dev\.connection_proxy: .?(\w*).? Disconnected -> (?:Wireless|Direct)"/> //this is a "device connected" regex override for iCUE 5
-		<add key="DisconnectedRegexOverride" value="\d{4}\-\d\d\-\d\d\s(?:\d\d\:)*\d\d.\d+ I cue\.dev\.connection_proxy: .?(\w*).? (?:Wireless|Direct) -> Disconnected"/> //this is a "device disconnected" regex override for iCUE 5
+		<add key="ConnectedRegexOverride" value='\d{4}-\d\d-\d\d\s(?:\d\d:?)+\.\d+\sI\scue\.dev\.connection_controller:\s"(?:\w|\s|\d)+"\sDisconnected\s->\s(?:Wireless)|(?:Direct)'/> //this is a "device connected" regex override for iCUE5
```

Comment:
since you're already touching it I'd keep the regex compatible with the current version as well (`connection_(controller|proxy)` and the quotes

Reply:
Done with non-capturing group `(?:)`
A normal group would output something, but the program looks just for the line in the log, no other info is being pulled out of the log, so no need for a normal group